utils/check-dependencies-exist.js

```diff
@@ -93,6 +93,10 @@ function formatErrorMessage(pkgFile, errors) {
               const column = errors[key][file].column;
               const message = errors[key][file].formatted;
 
+              if (!line && !column && !message) {
+                return `${file}\n${errors[key][file]}`;
+              }
```

Comment:
I've been meaning to add this for a while. There are errors that don't have line and column numbers. Parse errors fall in this category. This extra check here gives us useful output on parse errors.